apps/examples/testcase/ta_tc/systemio/itc/itc_uart.c

```diff
@@ -259,6 +259,38 @@ void itc_systemio_iotbus_uart_set_flowcontrol_p(void)
 	TC_SUCCESS_RESULT();
 }
 
+/**
+* @testcase         itc_systemio_iotbus_uart_loopback_p
+* @brief            check the uart loopback mode support
+* @scenario         check the uart loopback mode support
+* @apicovered       iotbus_uart_write, iotbus_uart_read
+* @precondition     initializes uart_context
+* @postcondition    closes uart_context
+*/
+void itc_systemio_iotbus_uart_loopback_p(void)
```

Comment:
How is this api different from below api "itc_systemio_iotbus_uart_write_read_p"?
We are performing same operations like opening same device, write, read and compare. The only difference I see is the text being written.

Reply:
yes, this is not different from mentioned API. 
I created this because sunghan wanted to create an API to check loopback so that is the only way to check loopback, so after discussing with sangam, i created this TC.